packages/dev-server/src/esbuild/plugins/loggingPlugin.ts

```diff
@@ -0,0 +1,40 @@
+import Log from '@expo/bunyan';
+import { Plugin } from 'esbuild';
+
+export default function loggingPlugin(logger: Log) {
+  const plugin: Plugin = {
+    name: 'expoLogging',
+    setup(build) {
+      build.onStart(() => {
+        logger.info(
```

Comment:
Add progress support if possible. Also figure out why progress loader isn’t showing in-app. 
Possibly related to the issue where logs don’t show up in flipper